common/scripts/upgrade.sh

```diff
@@ -4,6 +4,13 @@ readonly MAX_ERROR_LOG_COUNT=100
 export IS_SERVER=false
 export system_cluster=null
 
+__update_docker() {
```

Comment:
this will affect other installations of docker as well. In U14, C7, RHEL the docker version will already be upgraded in the `_helpers.sh` file

Reply:
we'll change that in next set of PR's